README.md

```diff
@@ -17,6 +17,13 @@ Requirements
 
 * [ZSH](http://zsh.sourceforge.net) 4.3 or newer
 
+Install
+------------------------------------------------------------------------------
+### Git
+* git clone https://github.com/zsh-users/zsh-history-substring-search.git
+
+### Brew
+* brew install zsh-history-substring-search
```

Comment:
Please (1) add a blank line after the subheading and (2) convert the list item into a code block.

Where does Homebrew install this plugin anyway?  Users needs to know so they can `source` it.